src/express/commands/rewind.js

```diff
@@ -0,0 +1,69 @@
+import { loadDevnetConfig, splitToArray } from '../common/config-utils'
+const {
+  runSshCommand,
+  runSshCommandWithReturn,
+  maxRetries
+} = require('../common/remote-worker')
+
+export async function rewind(num) {
+  //   num = number of blocks to rewind
+  if (num > 128) {
+    console.log(
+      '📍number of blocks to rewind should should be less than equal to 128, setting to 128'
```

Comment:
"should" is written twice. Also, can you leave an empty space before "number" and use capital N?